optin/puller/workflows/payout.py

```diff
@@ -135,14 +148,15 @@ async def main():
 	symbol_network = await client.node_network()
 	nem_network = NemNetwork.TESTNET if 'testnet' == args.network else NemNetwork.MAINNET
 
-	key_pair = get_key_pair(Path(args.hot))
-	print(f'building preparer around {symbol_network.public_key_to_address(key_pair.public_key)}')
+	funder_key_pair = get_key_pair(Path(args.hot))
+	funder_address = symbol_network.public_key_to_address(funder_key_pair.public_key)
+	funder_balance = await client.balance(funder_address)
 
-	transaction_preparer = TransactionPreparer(args.network, currency_mosaic_id, key_pair)
+	print(f'building preparer around {funder_address} with balance {funder_balance}')
 
-	node_time = await client.node_time()
-	deadline = node_time.add_hours(1).timestamp
-	print(f'using deadline {deadline}')
+	transaction_preparer = TransactionPreparer(args.network, currency_mosaic_id, funder_key_pair,)
+
+	deadline = await calculate_deadline(client)
```

Comment:
hmmm, I realized this might _not_ be greatest idea ...
with initial run with 400txes, well, when confirming tx 400th it might already be after deadline 😬